python/python/ert/config/schema_item.py

```diff
@@ -28,11 +30,38 @@ class SchemaItem(BaseCClass):
     _set_argc_minmax = ConfigPrototype("void config_schema_item_set_argc_minmax( schema_item , int , int)")
     _add_alternative = ConfigPrototype("void config_schema_item_add_indexed_alternative(schema_item , int , char*)")
     _set_deprecated = ConfigPrototype("void config_schema_item_set_deprecated(schema_item ,  char*)")
-
+    _valid_string = ConfigPrototype("bool config_schema_item_valid_string(config_content_type_enum ,  char*)", bind = False)
+    _sscanf_bool = UtilPrototype("bool util_sscanf_bool( char* , bool*)", bind = False)
```

Comment:
No - it is not prototyped anywere else. I'll leave it as is now - but I guess the good solution would be to establish a convention for prototyping of loose functions. Technically it is simple, it is mainly the organizing/namespacing I am uncertain of.